pkg/apis/core/validation/seed.go

```diff
@@ -277,6 +282,8 @@ func validateSeedNetworks(seedNetworks core.SeedNetworks, fldPath *field.Path, i
 	}
 	allErrs = append(allErrs, cidrvalidation.ValidateCIDROverlap(networks, false)...)
 
+	apiServerProxyRange := cidrvalidation.NewCIDR(v1beta1constants.ReservedRangeApiServerProxy, field.NewPath(""))
+	allErrs = append(allErrs, apiServerProxyRange.ValidateNotOverlap(networks...)...)
```

Comment:
While it is unlikely someone might have created a seed cluster overlapping with this range. How do we handle this?

Reply:
Release notes? There are more reserved ranges we don't check that could create all kinds of problems, e.g. multicast range. IMO gardener should have checked those from the beginning and forbid them.